packages/action-link-test/testActionLinks.js

```diff
@@ -0,0 +1,5 @@
+RocketChat.actionLinks.register('testFunct', function(origDbMsg, params) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["params" is defined but never used](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2674132099/issues/source?bid=3199634&fileBranchId=3333125#l1)